assets/scss/layout/_header.scss

```
@@ -1,4 +1,115 @@
-.header {
-  background: #fefae0;
-  font-size: 32px;
+@use 'sass:map';
+
+@include media-breakpoint-up(lg) {
+  .w-lg-30 {
+    width: 30%;
+  }
+  .w-lg-40 {
+    width: 40%;
+  }
+}
+
+.separator-line-top {
+  border-top: 0.8px solid $primary-200;
+}
+
+.custom-nav-link {
+  color: $neutral-700;
+  font-size: $font-size-sm;
+  display: block;
+  padding-top: 0.343rem;
+  padding-bottom: 0.343rem;
+}
+@include media-breakpoint-up(lg) {
+  .custom-nav-link {
+    font-size: $font-size-base;
+    padding-top: map.get($spacers, 1);
+    padding-bottom: map.get($spacers, 1);
+    padding-left: map.get($spacers, 3);
+    padding-right: map.get($spacers, 3);
+  }
+}
+
+.cart-toggle-btn {
+  color: $neutral-700;
+  padding: map.get($spacers, 2);
+  border-width: 0;
+  border-radius: 50%;
+  background-color: transparent;
+  &.active {
+    background-color: $primary-50;
+  }
+}
+@include media-breakpoint-up(lg) {
+  .cart-toggle-btn {
+    padding: map.get($spacers, 3);
+  }
+}
+.cart-badge {
+  font-size: $font-size-xxs;
+  transform: translate(-73.333%, -22.222%);
+}
+@include media-breakpoint-up(lg) {
+  .cart-badge {
+    font-size: $font-size-xs;
+    transform: translate(-50%, -40%);
+  }
+}
+
+.member-menu {
+  right: 0;
+  top: 4rem;
+  min-width: 12.25rem;
+}
+.member-menu-toggle-btn {
+  color: $neutral-700;
+  padding: map.get($spacers, 3);
+  border-width: 0;
+  border-radius: 50%;
+  background-color: $primary-50;
+  &:hover:focus,
+  :active {
+    background-color: $primary-50;
+  }
+  &.collapsed {
+    background-color: transparent;
+    &:hover:focus,
+    :active {
+      background-color: transparent;
+    }
+  }
+}
+.member-menu-link {
+  color: $neutral-700;
+  font-size: $font-size-sm;
+  display: block;
+  padding-top: 0.343rem;
+  padding-bottom: 0.343rem;
+}
```

```suggestion
.member-menu {
  right: 0;
  top: 4rem;
  min-width: 12.25rem;

  &-toggle-btn {
    color: $neutral-700;
    padding: map.get($spacers, 3);
    border-width: 0;
    border-radius: 50%;
    background-color: transparent;

    &:hover,
    &:active {
      background-color: $primary-50;
    }
  }

  &-link {
    color: $neutral-700;
    font-size: $font-size-sm;
    display: block;
    padding-top: 0.343rem;
    padding-bottom: 0.343rem;
  }
}
```

1. member-menu的巢狀結構可以這樣寫。
2. `&:hover:focus,:active`一樣有修改寫法。
3. 跟上面一樣設定原背景透明、移除focus跟collapsed的設定